pkg/cmd/workspace/util/creation_data.go

```diff
@@ -18,7 +18,10 @@ func GetCreationDataFromPrompt(workspaceNames []string, userGitProviders []serve
 	var providerRepo serverapiclient.GitRepository
 	var providerRepoUrl string
 	var err error
-	var confirmCheck bool
+	var workspaceName string
+	var primaryContainerImageUrl string
```

Comment:
Remove `Url` from name

Reply:
Removing "url" and "os" from all variables relevant to this PR